### DISCUSSION TITLE
OIIO logo

### DISCUSSION THREAD

**Discussion by Correct-Syntax** [OpenImageIO/oiio#3691] [Ideas]
Hello,

A while back I had mentioned in the mailing list about creating a real logo and possibly a (better) website for OpenImageIO.

I've started on an initial logo concept and wanted to get some direction from the OIIO maintainers (specifically @lgritz) before I went too far.

The concept I've come up with so far is based around pixels of an image. Inspired by [this](https://openimageio.readthedocs.io/en/latest/_images/grid-small.jpg) image.

## Questions

-  Is there any "official" OIIO color(s)? If not, do you have a preference?
- Am I on the right track here?

## 4 Initial concept ideas

*Note: The below logo concept images are purposely grayscale (no-color).*
**Edit:** the last image should have (1,0) instead of (0,1)

![oiio-logo-initial-concept-ideas](https://user-images.githubusercontent.com/60711001/204164660-19fc5008-1b71-498c-8b29-2715f7389712.png)

**Comment by lgritz** [comment]
Neato, yeah!

I like the direction and the concept. Random thoughts:

The "(0,1)" thing wasn't really working for me, mostly because it makes my brain uncontrollably nervous about people misunderstanding where the pixel origin is. :-) 

I like how the first image has the "pixels" kind of wrapping around the O. But I like the smaller pixels and their more rounded overall shape in the others. I wonder if those ideas can be combined -- smaller pixels, more rounded shape around the words. I also wonder if it might be flipped and have the pixel shapes on the upper left... excuse my crude low res text art, something like, schematically:

```
  * *                  * *
  * OpenImageIO        * OIIO

```

for the full-word and small versions. Hopefully you know what I'm getting at.

We have no "official colors", but I'm eager to hear you ideas about what they should be!

This is exciting. I was looking recently at the [OpenColorIO page](https://opencolorio.org), scroll down to the "Applications" section, and the OIIO, such as it is, is embarrassingly uninteresting. Looking at the [ASWF project logos](https://www.aswf.io/projects) also gives some feel for what other projects in our space are doing for logos.

**Reply by Correct-Syntax** [reply]
Alright, here's what I've come up with after a bunch of tweaking. 

(I think you are right. Having it on the left is better.)

How is this?

![oiio-logo-2-grayscale](https://user-images.githubusercontent.com/60711001/204429079-c564ec40-2919-44fb-9be7-b0d77a1e33af.png)

For colors I am thinking of a blue like #0583D6 but I'll continue to try different colors and post examples here.

**Comment by lgritz** [comment]
Yeah, that's exactly what I meant.

There's something a little odd about how the ragged edge doesn't have a consistent direction of curvature, if you know what I mean. Wondering if the bottom row should have 3 pixels wide instead of 2 so it doesn't appear so "ragged"? Maybe the ragged edge look is exactly what you intended, but I found it a little distracting at first glance.

But I totally like the way this is going.

Hey, in case it's not glaringly obvious, I'm no graphic designer, so if anybody else has opinions, please please to chime in. The project is nearing 15 years old and has never had a proper logo, so I can't help but be like a starving person who just got handed a piece of bread -- it's all going to taste great to me, so take my comments with a big grain of salt and don't be afraid to suggest something different.

**Comment by Correct-Syntax** [comment]
(Sorry for the delay. Three weeks went by really quick.)

> There's something a little odd about how the ragged edge doesn't have a consistent direction of curvature, if you know what I mean. 

I get what you mean. One of the variations below adds the pixel to the bottom row as you described.

> if anybody else has opinions, please please to chime in

Yes, it would be great to get some other opinions.

## More ideas

I created variations of the previous concept and also went along another path, experimenting with a different design style. The below image is the culmination of the ideas.

Let me know which one(s) of these stand out to you.

![logo ideas - 3@2x](https://user-images.githubusercontent.com/60711001/208324818-8cb189a7-aef1-4d89-8179-9485880579fb.jpg)

*In the last three ideas, "OIIO" is hidden in the logomark*

**Comment by lgritz** [comment]
The one with the blue, and the B&W version immediately above it, are exactly what I was imagining in the last round, I like them a lot.

The one immediately below the blue one, also, is pretty nice (and extrapolating to it having a splash of color, seems like it will be really attractive).

**Comment by Correct-Syntax** [comment]
Okay. Here's some colored versions of those two logo designs (these are in addition to the B&W versions).

If you have another particular color in mind I can give it a go with these designs as well.
 
![Logos 5@2x](https://user-images.githubusercontent.com/60711001/208528497-e8b0e760-1bf6-4eb5-a523-e42298a852b8.jpg)

**Reply by lgritz** [reply]
Very nice! I would say that all of the choices in this last round are great and equally acceptable to me, I'm not sure I have a strong preference. I hope to hear some other opinions, I'll post the link to this issue around and see if anybody has further feedback. But at this point, I don't think I'd have buyer's remorse on any of these.

**Comment by MrLixm** [comment]
Love them, awesome work.
I prefer the square-with-outline version on my side.

**Comment by carolalynn** [comment]
Nice work! I'm also not a graphic designer, so take what I say here with a grain of salt. I think these options are all great, and out of the ones presented, I like rows 2 & 3 of the latest sheet. Keeping it simple (sans gradient) will display the best in all situations. 

That said, one of the most useful parts of OCIO's logo has been the "bug" - the "C" part that can stand alone, or with the rest of the text. So I'd just make sure that in any kind of a brand guide, you specify whether or not the graphic part of this logo is ok for use, (say, in an application menu) without any text. It's totally fine if it's not, just something to think about.

**Comment by meshula** [comment]
I like it a lot, it inspired me to do a take on it ~

<img width="541" alt="image" src="https://user-images.githubusercontent.com/501238/208789799-26262e7d-89fb-451a-8782-98d5781c6ab7.png">

**Reply by meshula** [reply]
Your squares in the first proposal where they kind of wrapped around the logo reminded me of color chips from a color checker chart, and I liked how bold and graphic OIIO is. I found that two rows of chips and the Impact font version of OIIO fit into a square - I liked the idea that it works as a logo and an icon :)

**Comment by Correct-Syntax** [comment]
Thanks for everyone's input on this. 

@carolalynn22, that's a good point on the usage of the graphic part of the logo. I'll be sure to include that in OIIOs guidelines.

My personal choice is the square with the outline design as well, specifically the ones towards the bottom-right (the bold version). 

It seems there is a general consensus. 🎉 What do you think @lgritz? Would you like me to move on to getting the logo variations in the various formats and getting some brand guidelines together, or try something else?

**Comment by lgritz** [comment]
Yeah, bottom right ones FTW, sounds good to me!

**Comment by Correct-Syntax** [comment]
Alright, here's the Brand Guidelines draft and Logo assets. Let me know if you see anything incorrect, typos, etc, etc

## Brand Guidelines

[openimageio-brand-guidelines.pdf](https://github.com/OpenImageIO/oiio/files/10303980/openimageio-brand-guidelines.pdf)

## Assets

[openimageio-logo-assets-v1.0.zip](https://github.com/OpenImageIO/oiio/files/10303989/openimageio-logo-assets-v1.0.zip)

**Reply by meshula** [reply]
The logo in the pdf turned out really well, IMO :) 

I am wondering though, if a more universally available (and graphic) font might be chosen? The latin characters in Hind Bold are a bit thin to my eye, and given the prominence of the character O in the logotype, my preference would be to choose a font with a fuller O.

**Reply by meshula** [reply]
PS, I used to do typesetting as a business a long time ago and have residual fussiness around typography, please forgive my fussiness, I can't help myself :)

**Reply by Correct-Syntax** [reply]
Hello Nick (@meshula), thanks for sharing your thoughts on the logo font. Fussiness can be a good thing. 🙂 I'm not as experienced  on type so I appreciate getting your thoughts.

I mainly chose Hind because I liked how geometric, balanced, and modern-looking it is (the strokes have flat endings, etc) -matching with the Icon which is made of square shapes. 

That said, your comments inspired me to try some other fonts I hadn't considered like *Impact*. (See below)

I would leave any decision up to @lgritz, but my opinion would be that *Hind* represents the OpenImageIO library the best of any fonts thus far. 

![Logo font comparisons](https://user-images.githubusercontent.com/60711001/209686893-2476dc22-cee5-43b1-99d9-5c0bdf22b146.png)

Did you have a specific font in mind to try? (Attached is a zip of the Sketch file if you would like to experiment).

[oiio-logo-font-comparisions.zip](https://github.com/OpenImageIO/oiio/files/10309269/oiio-logo-font-comparisions.zip)

**Reply by meshula** [reply]
I can see your point about the geometric aspects.

If you convert the text to stroke, then we don't need to install fonts to use the logo.

Not having to install a font is more to the point ;)

**Reply by lgritz** [reply]
Oh boy, don't leave it to me. I defer the fine typography decisions to those who know and care more than I do. I think the concern about making people install fonts (especially rare or costly ones) is a good one.

FWIW, I'm not crazy about the "Impact" font, but any of the other three seem equally fine to me. My wife prefers the Open Sans (I rather like it, too, but not enough to try to talk you guys out of Hind if that's what you like best).

Also -- please shoot this down if it just shows my typography ignorance -- my eye keeps getting drawn to the space between the n and I, which to me looks noticeably larger than the spacing between any other adjacent letters (that is, it looks a bit like: Open ImageIO).

**Reply by meshula** [reply]
I was going to make some font suggestions and got distracted by this: https://www.fontfabric.com/font-tester/

[edit, accidentally posted a half sentence]
Hind has somewhat unique characteristics, with the ends of the lower case e and g distinguishing it from similar fonts like Helvetica.

I'm fine with Hind, as long as it's stroked in the logo artwork :)

**Reply by Correct-Syntax** [reply]
> If you convert the text to stroke, then we don't need to install fonts to use the logo.

For the .sketch logo comparison file I actually intentionally left the text as font based in case you wanted to experiment with them. 🙂

The Logo assets in the previously uploaded assets zip, though, are all converted to paths and the Hind font doesn't need to be installed.

**Reply by Correct-Syntax** [reply]
> My wife prefers the Open Sans (I rather like it, too, but not enough to try to talk you guys out of Hind if that's what you like best).

I would prefer and defer to your preference. Just let me know.

I think the Open Sans version is good as well. The primary differences are the shape of the g and slightly thinner letters. I think it comes down to what you want to communicate with the logo. I do see that with Open Sans, the thinner letters and more rounded shape comes across a bit less bold and possibly more friendly.

> my eye keeps getting drawn to the space between the n and I, which to me looks noticeably larger than the spacing between any other adjacent letters (that is, it looks a bit like: Open ImageIO).

I think you are right. I'll look into that.

**Comment by Correct-Syntax** [comment]
Ok. Which one?...
![Logos@2x](https://user-images.githubusercontent.com/60711001/210185931-9b8a83d8-8733-4360-bab1-27f42dd9bc75.png)

**Reply by meshula** [reply]
I find the very regular letter spacings in the second one make it more attractive overall.

**Reply by lgritz** [reply]
I very slightly prefer the bottom one, the weight is a little more to my liking. 

I like the style of the 'g' in the upper font, but not enough to push it ahead of the overall better feel of the lower one.

**Comment by Correct-Syntax** [comment]
# Brand guidelines

[openimageio-brand-guidelines.pdf](https://github.com/OpenImageIO/oiio/files/10341149/openimageio-brand-guidelines.pdf)

![promo-card](https://user-images.githubusercontent.com/60711001/210476514-f0b7a032-313f-42c4-9c70-cfb50c7d7317.png)

**Reply by lgritz** [reply]
Awesome! This is so great!

**Comment by Correct-Syntax** [comment]
# Assets

Here is the zip of the logo assets, .sketch files, the guidelines PDF, and a few promo graphics.

[oiio-brand-assets.zip](https://github.com/OpenImageIO/oiio/files/10347136/oiio-brand-assets.zip)
